src/index.js

```diff
@@ -7,8 +7,14 @@ const pump = require('pump')
 
 const promiseHyperdb = require('./promiseHyperdb')
 
-const HUB_URL = 'localhost:8080'
-const MASQ_APP_BASE_URL = 'http://localhost:3000'
+const MASQ_APP_BASE_URL = (function () {
+  switch (process.env.NODE_ENV) {
+    case ('development'):
+      return 'http://localhost:3000'
+    default:
+      return 'https://qwantresearch.github.io/masq-app/'
```

Comment:
We should not hardcode these values. Can you add a a config.js file where we define dev/pro urls ?
Maybe we will need to set others values too, so maybe create a folder with 2 configs: dev.config.js, and prod.config.js, and the correct one will be loaded depending on the `NODE_ENV`

Reply:
done